CHANGELOG_PENDING.md

```diff
@@ -17,6 +17,17 @@ program](https://hackerone.com/tendermint).
 
 ### FEATURES:
 
+- [rpc/lib] [\#4248](https://github.com/tendermint/tendermint/issues/4248) RPC client basic authentication support (@greg-szabo)
```

Comment:
move to `changelog.md`?

Reply:
also don't see version update

Reply:
https://github.com/tendermint/tendermint/pull/4298, sorry a bit brain dead right now